analyses/cell-type-ewings/template_notebooks/README.md

```diff
@@ -5,3 +5,17 @@ This folder contains any template notebooks that are rendered as part of a workf
 1. `cnv-workflow`: This folder contains all template notebooks used in `cnv-annotation.sh`.
 2. `auc-singler-workflow`: This folder contains all template notebooks used in `auc-singler-annotation.sh`.
 3. `clustering-workflow`: This folder contains all template notebooks used in `evaluate-clusters.sh`. 
+
+## Guide notebook for assigning and evaluating cell type annotations 
+
+The `celltype-exploration.Rmd` notebook is meant to be used as a guide for assigning and evaluating the final cell type annotations for each library in `SCPCP000015`. 
+Instructions for using this guide: 
```

Comment:
I think it would also be fine to say here "instructions are in the template notebook" instead of duplicating instructions, but that's up to you!